app/concerns/liquid_interpolatable.rb

```diff
@@ -189,6 +189,11 @@ def uri_expand(url, limit = 5)
       url
     end
 
+    # Rebase URIs contained in attributes in a given HTML fragment
+    def rebase_html(input, base_uri)
```

Comment:
I think it needs to include `html` in the name, because our use of Liquid is general text templating, not focused on HTML.